javascript/javascript1/week1/ez-namey.js

```diff
@@ -0,0 +1,11 @@
+const firstWords=["easy", "awesome", "amazing", "beautiful",
+ "business", "best", "bare", "good", "perfect", "primary"];
+const secondWords=["Corporation", "motivation", "plan", "start", "patners", 
+"office", "five", "hands", "home", "profit" ];
+
+const randomNumber = Math.floor(Math.random() * 10) + 0;
+
+const startupName=firstWords[randomNumber]+ ' ' +secondWords[randomNumber];
+const length=firstWords[randomNumber].length +secondWords[randomNumber].length;
```

Comment:
Perfect.
By the way, you can also use `lenght` with strings. for example:
```js
const length = startupName.length;
```
Taking into account that the empty space will be calculated.
⭐ you don't have to change it, I just wanted to mention another way.